### PR TITLE
chore: bump node 12.x to latest 12.22.2

### PR DESCRIPTION
**Pull request by trivikr**
*Issue #, if available:*
Node.js v12.20.1 was added in `ubuntu/standard/5.0`: https://github.com/aws/aws-codebuild-docker-images/blob/15e7890b1cdd8ec7391f2dbaf891a346d4b8c55b/ubuntu/standard/5.0/Dockerfile#L202

*Description of changes:*
bumps node 12.x to latest ~~v12.20.1~~ v12.22.2 in all standard Dockerfiles

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by subinataws** [comment]
@trivikr - Thanks for the PR. Note merging this PR on this repo unfortunately will not do anything for CodeBuild on AWS cloud. New versions will be picked up when the next image update for CodeBuild happens. If you need the latest node version, suggest using "n latest" in your buildspec to pull down the tip version in your build container.

File: al2/aarch64/standard/2.0/Dockerfile [hunk 1]
```diff
@@ -361,7 +361,7 @@ RUN set -ex \
     && docker-compose version
 
 # Node 12
-ENV NODE_12_VERSION="12.18.0"
+ENV NODE_12_VERSION="12.21.0"
```
Comment (trivikr) [inline al2/aarch64/standard/2.0/Dockerfile:364]:
```suggestion
ENV NODE_12_VERSION="12.22.2"
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/

File: al2/x86_64/standard/3.0/Dockerfile [hunk 2]
```diff
@@ -398,7 +398,7 @@ RUN set -ex \
     && pip3 install --no-cache-dir --upgrade setuptools wheel aws-sam-cli awscli boto3 pipenv virtualenv --use-feature=2020-resolver
 
 # Node 12
-ENV NODE_12_VERSION="12.19.1"
+ENV NODE_12_VERSION="12.21.0"
```
Comment (trivikr) [inline al2/x86_64/standard/3.0/Dockerfile:401]:
```suggestion
ENV NODE_12_VERSION="12.22.2"
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/

File: ubuntu/standard/4.0/Dockerfile [hunk 3]
```diff
@@ -217,7 +217,7 @@ RUN set -ex \
 
 #****************      NODEJS     ****************************************************
 
-ENV NODE_12_VERSION="12.19.1" \
+ENV NODE_12_VERSION="12.21.0" \
```
Comment (trivikr) [inline ubuntu/standard/4.0/Dockerfile:220]:
```suggestion
ENV NODE_12_VERSION="12.22.2" \
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/

File: ubuntu/standard/5.0/Dockerfile [hunk 4]
```diff
@@ -201,7 +201,7 @@ RUN set -ex \
 
 #****************      NODEJS     ****************************************************
 
-ENV NODE_12_VERSION="12.20.1"
+ENV NODE_12_VERSION="12.21.0"
```
Comment (trivikr) [inline ubuntu/standard/5.0/Dockerfile:204]:
```suggestion
ENV NODE_12_VERSION="12.22.2"
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/

**Review by leoherran-aws — APPROVED** [review]

**Pull request merged**